src/libData/AccountData/Transaction.cpp

```diff
@@ -147,6 +147,15 @@ const TransactionCoreInfo& Transaction::GetCoreInfo() const {
 
 const uint32_t& Transaction::GetVersion() const { return m_coreInfo.version; }
 
+// Check if the version is 1 or 2 - the only valid ones for now
+// this will look like 65538 or 65537
+bool Transaction::VersionCorrect() const {
+  auto version = DataConversion::UnpackB(this->GetVersion());
+
+  return !(version != TRANSACTION_VERSION &&
+           version != TRANSACTION_VERSION_ETH);
```

Comment:
Nitpick: it's perhaps a matter of style, but what about:
`return (version == TRANSACTION_VERSION || version == TRANSACTION_VERSION_ETH)`
Also version could be marked as const.

Reply:
👍